app/build.gradle

```diff
@@ -20,6 +20,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
I've encountered a bizarre issue, when the .idea/misc.xml's languageLevel tag keept changing JDK to 1.7, so I explicitly required version 1.8 to avoid further changes.
But sadly, I have no idea why this happened in the first place.